src/toolkit/res_buf.h

```diff
@@ -163,6 +172,14 @@ class ResBuf {
     return Pop(qty);
   }
 
+  /// Pops the specified amount of material, re-packed into the the 
```

Comment:
I'm not sure what this comment refers to now? Is it in the wrong place or did we delete the function that this was about?